pkg/system/phase1_verifying.go

```diff
@@ -275,25 +275,28 @@ func (r *Reconciler) checkBucketLoggingPVC() error {
 			"'Guaranteed' BucketLogging requires a Persistent Volume Claim (PVC) with ReadWriteMany (RWX) access mode. Please specify the 'BucketLoggingPVC' to ensure guaranteed logging")
```

Comment:
I would just add here the check.
```suggestion
     if r.NooBaa.Spec.BucketLogging.BucketLoggingPVC == nil {
            if ceph-fs exist -> good -> return nil
            else throw the persistent error
     } 
```
rest of the code is the same we know for BucketLoggingPVC is not nil

Reply:
This should be the best way, I will update. Thanks